org_status/org_status.py

```diff
@@ -87,14 +89,21 @@ def main():
     for Host, org in generate_fetch_jobs(args.orgs):
```

Comment:
We need to support a host only checking mode without the use having to pass any org to check. The best hacky way to currently achieve this would be to add a conditional block which append say `'coala'` to `args.orgs` when `args.hosts_only` is True and `len(args.orgs) == 0`. Note that `len(args.orgs)` cannot be zero as of now but you can change that by replacing `+` to a `*` in https://github.com/ksdme/org-status/blob/4e2ccd559468729302dbc6c63be369293ee57797/org_status/org_status.py#L72. Keep an eye out for side effects this may cause.